samples/nrf9160/modem_shell/README.rst

```diff
@@ -0,0 +1,455 @@
+.. _modem_shell_application:
+.. highlight:: console
```

Comment:
I've been trying to change this back and forth between deleting this line, having sh, c or console to try and find pygments lexer that wouldn't do any highlighting of the examples, or literal blocks in the final document but haven't found so far. Problem with this console is that it's light grey and could be darker. In addition, bootup text in Testing section has highlighting.
@peknis Can you check what would be the best output for example shell command highlighting? Leaving this line out will set default which is python3 (or some python) and that's highlight the end of curl commands that have python's comment character (#) in the command.

Reply:
Before each code snippet:
.. code-block:: console
   
   add the command here

Reply:
`highlight `should do the same for entire document (until next `highlight`) as code-block does for one literal block. My question is whether there is a better style that `console `for our purpose. Ideally it would be something that doesn't highlight any parts of the text but even google doesn't help me finding such lexer as python based lexers (python, python3, etc.) seem to be defaults.

Reply:
Will look into this. I'll get back when I have a better answer.

Reply:
Possibly, .. code-block:: none leaves the codesnippet without any highlighting.

Reply:
I'll give `none `value a try now. Somewhere in web I read that it would mean default which is `python3 `but let's see. It would make sense to have a `none `value with now highlighting.

Reply:
You can see how none shows up at the moment. Looks fine except for the bootup part. I guess that looked fine with python3 so I'll try that out there.

Reply:
I'm using currently `.. highlight:: none` in the beginning of the file so all literal blocks will use this unless otherwise specified. This seems to work fine now.

Reply:
This looks good now.